src/srf/surface.h

```diff
@@ -156,7 +156,7 @@ class SBezierLoopSet {
     static SBezierLoopSet From(SBezierList *spcl, SPolygon *poly,
                                double chordTol,
                                bool *allClosed, SEdge *errorAt,
-                               SBezierList *openContours);
+                               SBezierLoopSet *openContours);
```

Comment:
I don't quite understand why this is needed. Could you elaborate on it? That would also make a good commit message.

Reply:
Due to their different style, the engravings end up in the list of open contours during `FindOuterFacesFrom`, regardless of whether they form a closed contour or not. Considering that openContours is a list of segments, open loops have to be added per segment, thereby losing their connectivity. For example, an engraved circle will be exported as 4 separate arcs (at least for SVG). To fix this, I had to change the type of open contours to a loopset, which does retain connectivity.

Reply:
I'm still confused. Aren't open contours, by definition, not bezier loops?

Reply:
I guess. Though `SBezierLoop::FromCurves` returns open loops and there is `SBezierLoop::IsClosed()` to tell if a loop is closed. So I assumed that it's okay for loops to be open. And in every other aspect, that class does exactly what I need from it. If you want, I can rename `SBezierLoop` to `SBezierContour`, to make it more consistent.

Reply:
Oh, you're right. SBezierLoop doesn't have to be closed. That's OK then.